### PR TITLE
cube: Move android files to cube directory

### PR DESCRIPTION
**Pull request by juan-lunarg**
Despite being in the "common" folder these files are only used by vkcube.

### CODE REVIEW

File: cube/android/README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Copy contents of parsed_header into generate_vulkan_wrapper.py between the secti
 Then run the script:
 
     popd
-    ./generate_vulkan_wrapper.py ../common/vulkan_wrapper.h
-    ./generate_vulkan_wrapper.py ../common/vulkan_wrapper.cpp
+    ./generate_vulkan_wrapper.py ../cube/android/vulkan_wrapper.h
+    ./generate_vulkan_wrapper.py ../cube/android/vulkan_wrapper.cpp
```
Comment (charles-lunarg) [inline cube/android/README.md:26]: So is android the only thing that uses vulkan_wrapper? I was half under the impression that it was used for something else (possibly not in the repo?)

Comment (juan-lunarg) [reply cube/android/README.md:26]: As far as I can tell these files are only used for android for vkcube. I don't see anything indicating otherwise.

**Comment by juan-lunarg** [comment]
GHA passes on fork

**Comment by juan-lunarg** [comment]
https://github.com/juan-lunarg/Vulkan-Tools/actions/runs/6472774914

**Pull request merged**